### PR TITLE
[Doc] Add guides for AMQP1_0 connector

### PR DESCRIPTION
**Pull request by Anonymitaet**

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@gaoran10 @Jennifer88huang @Huanli-Meng could you please help review? Thanks

**Comment by gaoran10** [comment]
@timmyyuan Hi, yuan ting, could you please take a look at the test problem? Thanks.

```
Checking url https://github.com/streamnative/pulsar-io-amqp-1-0/tree/branch-2.7.1/src/main/java/org/apache/pulsar/ecosystem/io/random
Checking url https://www.apache.org/licenses/LICENSE-2.0
Checking url https://github.com/streamnative/pulsar-io-amqp-1-0/releases/download/v2.7.1.1/pulsar-io-amqp1_0-2.7.1.1.nar
Checking url https://streamnative.io
Invalid url https://github.com/streamnative/pulsar-io-amqp-1-0/tree/branch-2.7.1/src/main/java/org/apache/pulsar/ecosystem/io/random
Invalid url https://github.com/streamnative/pulsar-io-amqp-1-0/releases/download/v2.7.1.1/pulsar-io-amqp1_0-2.7.1.1.nar
The required fields owner_name,owner_img are not in the file /home/runner/work/pulsar-hub/pulsar-hub/prm/connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md.
Error: Process completed with exit code 1.
```

File: connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
+---
+description: The AMQP1_0 sink connector pulls messages from Pulsar topics and persists messages to AMQP brokers.
+author: ["StreamNative"]
+contributors: ["StreamNative"]
+language: Java
+document: 
+source: "https://github.com/streamnative/pulsar-io-amqp-1-0/tree/branch-2.7.1/src/main/java/org/apache/pulsar/ecosystem/io/random"
```
Comment (gaoran10) [inline connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md:7]:
```suggestion
source: "https://github.com/streamnative/pulsar-io-amqp-1-0/tree/branch-2.7.1/io-amqp1_0-impl/src/main/java/org/apache/pulsar/ecosystem/io/amqp"
```

File: connectors/amqp-1-0-source/2.7.1.1/amqp-1-0-source.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
+---
+description: The AMQP1_0 source connector receives messages from AMQP service and writes messages to Pulsar topics.
+author: ["StreamNative"]
+contributors: ["StreamNative"]
+language: Java
+document: 
+source: "https://github.com/streamnative/pulsar-io-amqp-1-0/tree/branch-2.7.1/src/main/java/org/apache/pulsar/ecosystem/io/random"
```
Comment (gaoran10) [inline connectors/amqp-1-0-source/2.7.1.1/amqp-1-0-source.md:7]:
```suggestion
source: "https://github.com/streamnative/pulsar-io-amqp-1-0/tree/branch-2.7.1/io-amqp1_0-impl/src/main/java/org/apache/pulsar/ecosystem/io/amqp"
```

**Comment by Anonymitaet** [comment]
@gaoran10 I've incorporated your comments, could u PTAL?

File: connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+|------|----------|----------|---------|-------------|
+| `protocol` |String| true | "amqp" | The AMQP protocol. |
+| `host` | String| true | " " (empty string) | The AMQP service host. |
+| `port` | int |true | 5672 | The AMQP service port. |
+| `username` | String|false | " " (empty string) | The username used to authenticate to ActiveMQ. |
+| `password` | String|false | " " (empty string) | The password used to authenticate to ActiveMQ. |
+| `queue` | String|false | " " (empty string) | The queue name that messages should be read from or written to. |
+| `topic` | String|false | " " (empty string) | The topic name that messages should be read from or written to. |
+| `activeMessageType` | String|false |0 | The ActiveMQ message simple class name. |
+| `onlyTextMessage` | boolean | false | false | If set to `true`, it requires that the AMQP message type is `TextMessage`. Pulsar consumer can consume the messages with schema ByteBuffer. |
```
Comment (Huanli-Meng) [inline connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md:69]:
```suggestion
| `onlyTextMessage` | boolean | false | false | If it is set to `true`, the AMQP message type must be set to `TextMessage`. The Pulsar consumer can consume the messages with schema ByteBuffer. |
```

File: connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+PULSAR_HOME/bin/pulsar-admin sinks create \
+--name amqp1_0-sink \
+--archive pulsar-io-amqp1_0-{version}.nar \
+--classname org.apache.pulsar.ecosystem.io.amqp.AmqpSink \
+--sink-config-file amqp-sink-config.yaml
+```
+
+## Use as built-in connector
+
+You can make the AMQP1_0 sink connector as a built-in connector and use it on standalone cluster, on-premises cluster, or K8S cluster.
```
Comment (Huanli-Meng) [inline connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md:136]:
```suggestion
You can make the AMQP1_0 sink connector as a built-in connector and use it on the standalone cluster, on-premises cluster, or K8S cluster.
```

File: connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+    ```
+    PULSAR_HOME/bin/pulsar-admin sinks create \
+    --sink-type amqp1_0 \
+    --sink-config-file amqp-sink-config.yaml \
+    --name amqp1_0-sink
+    ```
+
+### K8S cluster
+
+This example demonstrates how to create an AMQP1_0 sink connector on a K8S cluster.
```
Comment (Huanli-Meng) [inline connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md:345]: in a cluster or on a cluster? Please keep consistent throughout the doc.

**Review by Huanli-Meng — COMMENTED** [review]
leave my comments. PTAL

File: connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md (hunk 4 above)
Comment (Anonymitaet) [reply connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md:136]: here we talk about things in general, so do not use articles

**Comment by Anonymitaet** [comment]
@Huanli-Meng  I've incorporated your comments, could u PTAL?

**Pull request closed without merging**